README.md

```diff
@@ -1,24 +1,39 @@
-![conduit][logo]
+![Linkerd2][logo]
 
 [![Build Status][ci-badge]][ci]
 [![GitHub license][license-badge]](LICENSE)
 [![Slack Status][slack-badge]][slack]
 
-:balloon: Welcome to Conduit! :wave:
+:balloon: Welcome to Linkerd2! :wave:
 
-Conduit is an ultralight service mesh for Kubernetes. It features a minimalist
-control plane written in Go, and a native proxy data plane written in
-[Rust][rust] that boasts the performance of C without the heartbleed.
+Note: this project is currently in the middle of a migration from the old name (Conduit) to the new name (Linkerd2). While this transition is ongoing, there will references to Conduit that don't make a lot of sense. Not to worry, we're working on it! For more information, check out the [announcement][announcement].
 
-Conduit is **alpha**. It is capable of proxying all TCP traffic, including
-WebSockets and HTTP tunneling, and reporting top-line metrics (success rates,
+Linkerd2 is an ultralight *service mesh*, designed to make modern applications
+safe and sane by transparently adding service discovery, load balancing, failure
+handling, instrumentation, and routing to all inter-service communication.
+
+Linkerd2 (pronouned "linker-DEE-two") acts as a transparent
+HTTP/gRPC/thrift/tcp/etc proxy, and can be deployed alongside existing
+applications regardless of what language they're written in. It works with many
+common protocols and utilizes Kubernetes as a backend for service discovery.
+
+It is separated into two major components: the control plane and the data
+plane. The control plane interacts with the service discovery backend,
+orchestrates the data plane and is written in [Go][golang]. The data plane runs
+alongside existing applications, provides the proxy that manages traffic itself
+and is written in [Rust][rust].
+
+Currently, Linkerd2 is capable of proxying all TCP traffic, including
+WebSockets and HTTP tunneling, along with reporting top-line metrics (success rates,
 latencies, etc) for all HTTP, HTTP/2, and gRPC traffic.
 
+Linkerd is hosted by the Cloud Native Computing Foundation ([CNCF][cncf]).
```

Comment:
The `cncf` link doesn't exist.

![image](https://user-images.githubusercontent.com/9226/42773115-28199ff2-88e1-11e8-8386-b62bd3ee2683.png)